ocaml/xapi/xapi_host.ml

```diff
@@ -566,9 +566,28 @@ let evacuate ~__context ~host =
     match plan with
     | Migrate host ->
         ( try
+            ( if network <> Ref.null then
+                let hosts = Db.Host.get_all ~__context in
+                List.iter
+                  (fun host ->
+                    ignore
+                    @@ Xapi_network_attach_helpers
+                       .assert_valid_ip_configuration_on_network_for_host
+                         ~__context ~self:network ~host)
+                  hosts
+            ) ;
+            let options =
+              ("live", "true")
+              ::
+              ( if network <> Ref.null then
+                  [("network", Ref.string_of network)]
+              else
+                []
+              )
+            in
```

Comment:
```suggestion
            let with_network_option =
              if if network <> Ref.null then
                [("network", Ref.string_of network)]
              else
                []
            in
            let options = ("live", "true") :: with_network_option in
```

Reply:
Why is there 2 `if`s in your suggestion?

Reply:
it's a typo